user_tools/src/spark_rapids_tools/cmdli/argprocessor.py

```diff
@@ -25,14 +25,8 @@
 from pydantic.dataclasses import dataclass
 from pydantic_core import PydanticCustomError
 
-<<<<<<< HEAD:user_tools/src/pyrapids/cmdli/argprocessor.py
 from pyrapids.cloud import ClientCluster
```

Comment:
Incorrect imports
Should point to `spark_rapids_pytools`

Reply:
Thanks! Updated now.